qubes/storage/reflink.py

```diff
@@ -391,27 +416,31 @@ def _create_sparse_file(path, size):
         tmp.truncate(size)
         LOGGER.info('Created sparse file: %s', tmp.name)
 
+def _update_loopdev_sizes(img):
+    ''' Resolve img; update the size of loop devices backed by it. '''
+    needle = os.fsencode(os.path.realpath(img)) + b'\n'
+    for sys in glob.iglob('/sys/block/loop[0-9]*/loop/backing_file'):
```

Comment:
Clobbering `sys` module name (even if not imported here) is asking for troubles.

Reply:
Eek, yes.